### DISCUSSION TITLE
AssociationOverrides not available as PHP 8 attribute?

### DISCUSSION THREAD

**Discussion by stephanvierkant** [doctrine/doctrine2#8881] [Support Questions] [answered]
I found [this question](https://stackoverflow.com/questions/68605726/how-to-override-doctrines-field-association-mappings-in-subclasses-when-using-p) and I can't find why `AssociationOverrides` hasn't been changed to support PHP 8 attributes.

Anyone knows why this hasn't been changed? Or can I create a PR?

**Comment by beberlei** [comment, marked as answer]
Its not supported because nested attributes are not possible. We will add this for 8.1
